backend/clubs/admin.py

```diff
@@ -370,6 +371,7 @@ class QuestionAnswerAdmin(admin.ModelAdmin):
 admin.site.register(MembershipInvite, MembershipInviteAdmin)
 admin.site.register(Profile, ProfileAdmin)
 admin.site.register(QuestionAnswer, QuestionAnswerAdmin)
+admin.site.register(RecurringEvent)
```

Comment:
You might want to override the `__str__` function in the `RecurringEvent` model, otherwise the admin won't be too useful. The alternative would be to add more columns to the admin using a custom `ModelAdmin` object, which would be cleaner but more work.

I would recommend the string override returning a format like `{event name}: {start date} - {end date} ({occurences} times)`, where the event name is the name for the earliest instance, the start date is the date for the earliest instance, and the end date is the date for the latest instance (ideally, these would be separate columns in the admin).